q2_cutadapt/tests/test_demux.py

```diff
@@ -200,6 +200,27 @@ def test_batch_size_odd_number_of_samples(self):
         # obs_untrimmed should be empty, since everything matched
         self.assert_untrimmed_results(b'', obs_untrimmed_art)
 
+    def test_min_length(self):
+        metadata = CategoricalMetadataColumn(
+            # The third barcode is meant to completely remove the only GGGG
+            # coded sequence
+            pd.Series(['AAAA','CCCC','GGGGACGTACGT'], name='Barcode',
```

Comment:
```suggestion
            pd.Series(['AAAA', 'CCCC', 'GGGGACGTACGT'], name='Barcode',
```

This missing whitespace is reported when you run `make lint` on this project (you would normally see this report from our travis runs, too, but our travis builds are currently down. To learn more about what `make lint` runs, you can open the `Makefile` up and take a peek at the commands it runs. To learn more about _how_ those tools are installed (for running the `make lint` directive), you can pop open `.travis.yml` to see (among other things) what gets installed in a typical travis testing environment.